package.json

```diff
@@ -16,17 +16,17 @@
   ],
   "scripts": {
     "build": "tsc",
-    "test": "jest",
+    "test": "jest && cd website && yarn test",
     "test:update": "jest -u",
     "test:ci": "jest --runInBand",
     "test:watch": "jest --watch",
     "postinstall": "opencollective-postinstall || exit 0",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
-    "prettify": "prettier --single-quote --trailing-comma=es5 --write './**/*.md'",
     "clean-install": "rimraf node_modules && yarn",
     "changelog": "auto-changelog -p",
-    "docs": "cd website && yarn start",
-    "build:docs": "ts-node scripts/docgen/index.ts"
+    "docs": "yarn docs:build && cd website && yarn start",
+    "docs:lint": "prettier --single-quote --trailing-comma=es5 --write '**/*.md'",
```

Comment:
There's a `prettier` config at the root 
```js
{
  "singleQuote": true,
  "trailingComma": "es5",
  "endOfLine": "auto"
}

```

Reply:
So it should simply be "docs:lint": "prettier '**/*.md'" right?

Reply:
@pranshuchittora seems like it doesn't work with the .prettierrc file. This is only for the editor to prettify the code. for the code to be formatted we have to add flags.

Reply:
Yep